components/alert.css

```diff
@@ -0,0 +1,27 @@
+.alert-box {
+	height: 4rem;
+	font-size: 1.7rem;
+	margin: 1rem;
+	display: flex;
+	align-items: center;
+	padding: 0.5rem 2.5rem;
```

Comment:
margin and padding can go to seperate files  for some global spacing styles.

Reply:
I have them in the utility file. These are individual paddings for the specific element.